### DISCUSSION TITLE
Tabs for opening multiple files

### DISCUSSION THREAD

**Discussion by oldman20** [gsantner/markor#2226] [Ideas] [answered]
![Screenshot_20240218-233730_Google Play Store](https://github.com/gsantner/markor/assets/39668756/d077b261-ff5e-4c64-8599-201c4c89344b)
I need this function, and others too, surely

**Comment by gsantner** [comment]
Probably interesting to some users. Please go ahead and implement it / a first draft of it. Thank you!

**Comment by harshad1** [comment, marked as answer]
We have the ability to open multiple files in separate windows. I think switching between them using android navigation is just as easy as tabs.
